prime-angular-frontend/src/app/modules/adjudication/pages/license-classes-maintenance-page/license-classes-maintenance-page.component.ts

```diff
@@ -69,9 +70,10 @@ export class LicenseClassesMaintenancePageComponent implements OnInit {
             collegeName: college.name,
             licenseCode: collegeLicense.licenseCode,
             discontinued: collegeLicense.discontinued,
+            collegeLicenseGroupingCode: collegeLicense.collegeLicenseGroupingCode,
             ...license
           } as LicenseMaintenanceConfig;
-        }).sort(this.utilsService.sortByKey<LicenseMaintenanceConfig>('weight'))
+        }).sort(this.utilsService.sortByKey<LicenseMaintenanceConfig>('collegeName'))
```

Comment:
It's unclear why there is a need to change ordering?

Reply:
well, it is partially of my bad. The weight values are not set against whole list and some are all the same value, 10. Because of that, sorting by weight is wrong for sure. I tried to remove the sorting all together and it seems working alright. I will keep no sorting until business wait it to sort in a different way.

Reply:
OK, yeah, the order has changed compared to `demo` but it is still grouped together logically 👍